crates/snapbox/src/data.rs

```diff
@@ -107,6 +175,8 @@ impl Data {
                 Ok(data)
             }
             DataInner::Text(data) => Ok(data),
+            #[cfg(feature = "json")]
+            DataInner::Json(value) => Ok(serde_json::to_string_pretty(&value).unwrap()),
```

Comment:
It looks like `into_string` isn't really needed anymore and that we can move `make_text` to using `try_coerce`

This should be done in a PR before this one

Reply:
I opened #125 to address this